docs/src/prototypes/popups/GridImagePicker/GridImagePickerItem.tsx

```diff
@@ -0,0 +1,46 @@
+import { Image, Button } from '@stardust-ui/react'
+
+import * as React from 'react'
+import * as _ from 'lodash'
+
+export interface GridPickerItemProps {
+  as?: keyof React.ReactHTML
+  title?: string
+  imageSrc: string
+  onClick?: (e: React.SyntheticEvent, props: GridPickerItemProps) => void
+}
+
+const imageStyles = {
+  width: '100%',
```

Comment:
Have you tried using the fluid prop for the image, instead of these styles?

Reply:
I used ```fluid```, you're right, this one is redundant